src/cli/voters

```diff
@@ -0,0 +1,111 @@
+111122ww54CKibkmuS9dGZm98uAAu48wtrgBLyH4bsTdGhWCwucEp
```

Comment:
Hmm... Checking in the voter list... Usually derived stuff isn't checked in.

Perhaps I missed an explanation for this exception